src/wp-includes/functions.php

```diff
@@ -8274,15 +8274,23 @@ function wp_get_default_update_php_url() {
  *
  * @since 5.1.0
  * @since 5.2.0 Added the `$before` and `$after` parameters.
+ * @since 6.4.0 Added the `$echo` parameter.
  *
  * @param string $before Markup to output before the annotation. Default `<p class="description">`.
  * @param string $after  Markup to output after the annotation. Default `</p>`.
+ * @param bool   $echo   Markup should echo if true. Default `true`.
+ *
+ * @return string|void
  */
-function wp_update_php_annotation( $before = '<p class="description">', $after = '</p>' ) {
+function wp_update_php_annotation( $before = '<p class="description">', $after = '</p>', $echo = true ) {
```

Comment:
FYI in case you hadn't seen it: [Here's what I did](https://github.com/WordPress/wordpress-develop/pull/5149/files#diff-03c8ac6c0d90d91bd93d3aa0982efa4248f993cf4f009c225db938f2d64f505e) for these type of notices.

Happy for this function to be updated though if that makes more sense.

Reply:
If there's an existing getter, that probably makes more sense.

Reply:
On looking, the getter isn't equivalent, so I think this solution may be better.